CMakeLists.txt

```diff
@@ -211,6 +211,15 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMake")
 
+# add an option for building the test suite
+sfml_set_option(SFML_BUILD_TEST_SUITE FALSE BOOL "TRUE to build the SFML test suite, FALSE to ignore it")
```

Comment:
Should this be defined only if `SFML_CONFIGURE_EXTRAS` option is set?

Reply:
Yes it is

Reply:
Can't options also be declared later on, so that `if` statement still work in CMake anyways?

Meaning, there's no need to move the option

Reply:
@eXpl0it3r: you are correct, I will move it back.